docs/glossary.md

```diff
@@ -0,0 +1,87 @@
+# Glossary
+
+### Attribute
+A characteristic of a material, process, or measurement. Attributes can be a GEMD property, condition, or parameter.
+
+### Attribute Template
+Attribute templates define a domain concept (e.g. density, yield strength), and include bounds to define acceptable values for a given attribute.
+
+### Bounds
+Bounds are used in Object and Attribute templates. Object templates define the least constrained bounds, and attribute templates can constrain bounds to a tighter range or set.  Bounds may be real or categorical.
+
+### Categorical Bounds
+Bounds that are defined by a list of categories.
+
+### Condition Attribute
+Environmental variables (typically measured) that may affect a process or measurement, such as temperature or pressure.
+
+### Conditions
+Environmental aspects of a process or measurement.
```

Comment:
Why both?

Reply:
Hm I'm not sure I follow here, I'm looking at 

> Conditions are the environmental variables (typically measured) that may affect a process or measurement: e.g. Temperature, Pressure.
 from https://citrineinformatics.github.io/gemd-docs/specification/attributes/#property-condition-parameter
 
 what nuance am I missing?

Reply:
Ohh I see, the condition vs condition attribute. I agree, I'll axe conditions 👍